README.md

```diff
@@ -2,4 +2,100 @@
+
+  - FSRS available as an opt-in feature replacing the default SM-2 algorithm.
+  - In addition to feature metioned above, this [add-on](https://ankiweb.net/shared/info/759844606) helps with reshedule, postpone, advance, and balance cards, as well as enabling Easy Days scheduling, among other features.
+
+- #### MaiMemo (https://www.maimemo.com/)
+
+  Chinese mobile flashcard application for studying English.
+
+### Note-taking
+
+- #### RemNote (https://www.remnote.com/)
+
+  Multiplatform note-taking application with flashcard creation in single click. Has an active community with student-made exam materials on various topics. Available for offline and online usage.
+
+  RemNote integrated FSRS v4.5 to its scheduling system starting from [RemNote 1.16](https://feedback.remnote.com/changelog/remnote-1-16-ultimate-spaced-repetition).
+
+- #### Obsidian (https://obsidian.md/)
+
+  A personal notes, journaling, knowledge base, and project managament application with graph view and canvas diagram note. Connect your notes via hyperlinks and install many community plugins made for Obsidian.
+
+  FSRS available as extension for Obsidian: [obsidian-spaced-repetition-recall](https://github.com/open-spaced-repetition/obsidian-spaced-repetition-recall).
+
+### Others
+
+- #### Chessbook (https://chessbook.com/)
+
+  Mobile and web chess repertoire study application, combining multiple sources from YouTube, Lichess, ChessBase, and books to create a custom personal chess repertoire.
+
+  Chessbook overhauled its spaced repetition system using FSRS v4.5 and saw [20-30% cut down on reviews count](https://x.com/chessbookcom/status/1805137108991946775).
+
+## Related
+
+### Algorithm
+
+- #### Leitner System
+
+  Leitner sytem sorts flashcard into groups according to how well the learner knows each one in Leitner's learning box. The learners try to recall the solution written on a flashcard. If they succeed, they send the card to the next group. If they fail, they send it back to the first group. Each succeeding group has a longer period before the learner is required to revisit the cards. (More: [Wikipedia](https://en.wikipedia.org/wiki/Leitner_system))
+
+- #### SuperMemo algorithms
+
+  SuperMemo is the first software to apply computer-based spaced repetition algorithm.
+
+  - SM-0: [The birthday of spaced repetition: July 31, 1985](https://supermemo.guru/wiki/The_birthday_of_spaced_repetition:_July_31,_1985)
+  - SM-2: [Application of a computer to improve the results obtained in working with SuperMemo method](https://super-memory.com/english/ol/sm2.htm)
+  - SM-5: [First fast-converging spaced repetition algorithm: Algorithm SM-5](https://supermemo.guru/wiki/First_fast-converging_spaced_repetition_algorithm:_Algorithm_SM-5)
+  - SM-17: [Algorithm SM-17](https://supermemo.guru/wiki/Algorithm_SM-17)
+
+- #### ACT-R
```

Comment:
Is there anything worth noting about ACT-R and DASH?
Or would you say FSRS already superseded them in terms of functionalities and concepts?
@Expertium

Reply:
Tbh, I don't have a very good understanding of how they work. The only thing I know is that ACT-R doesn't make a distinction between failed and passed reviews, which is weird.